ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1102,19 +1135,25 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             <TooltipTrigger content={
               errors && <ul>
                 {errors.name && <div>{errors.name}</div>}
-                {errors.billingAccountName && <div>You must select a billing account</div>}
-                {errors.primaryPurpose && <div>You must choose at least one primary research
+                {errors.billingAccountName && <div>
+                  You must select a billing account</div>}
+                {errors.primaryPurpose && <div> You must choose at least one primary research
                   purpose (Question 1)</div>}
-                {errors.anticipatedFindings && <div>Answer for <i>What are the anticipated findings
+                {errors.otherPrimaryPurpose && <div> Other primary purpose should be of atleast 500 characters</div>}
```

Comment:
Shouldn't this error message say `at most 500 characters`? Since `rp.otherPurposeDetails.length <= 500`

Reply:
yes yeah i'll change it for all the relevant fields